psi-ms.obo

```diff
@@ -22682,6 +22682,49 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
 relationship: has_column: UO:0000191 ! fraction
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000072
+name: observed mass accuracy
+def: "Observed mass accuracy in ppm, calculated by 1E6 x (observed m/z - theoretical m/z)/theoretical m/z of a selected peptide spectrum." [PSI:MS]
```

Comment:
instead of "peptide spectrum" use "selected identified spectrum"?

Reply:
yes, that's better, thanks!

Reply:
How about ".. of a selected and identified precursor ion."? Though I noticed that 'precursor ion' is a term in the CV that is marked obsolete. Does anyone know why?

Reply:
... formally, 
1) an observed mass accuracy does not have to be restricted to a precursor ion; it can be applied to any m/z. 
2) a "mass" accuracy is not 100% correct for a m/z value, but here one can accept this extrapolation.
3) then, observed... we observe a m/z value, an ion chromatogram; but sometime, when an ion is "annotated" by the processing sw, for instance a b4-ion, the accuracy is calculated and not observed

that's why I prefer "of a selected ion in a mass spectrum" . As we are in a MS CV, the term refers to an ion in a mass spectrum. It is a selected ion as it is not a randomly chosen one. I can aggree to add "identified" as without "identification", no theoretical mass could be calculated. And ion is preferred to m/z here as it can include calculated charge-reduced ions obtained from a charge reduction of a m/z charge cluster.

Therefore ok for "of a selected and identified ion in a mass spectrum".

Reply:
Resolved?
https://github.com/HUPO-PSI/psi-ms-CV/blob/2361016e2d99d06cc5ccf57878a63474cf77073a/psi-ms.obo#L22685-L22693

Reply:
Thanks, @pabinz , can you mark the PR as accepted if you think the new terms are ok now?

Reply:
ok for me, thanks